### DISCUSSION TITLE
How to remove unused services from UI

### DISCUSSION THREAD

**Discussion by MuJianxuan** [apache/skywalking#9390] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/50062346/181155698-fa9e2313-2063-4d7c-805c-223aaa92ec35.png)

I forgot to modify the service name at first. 

After I changed it, the original became useless, I wonder how I could delete these kind of data. 

Please advice. 

Original:
我一开始忘记改名字了
然后我调整后，这个就成垃圾数据了，这怎么删除啊
麻烦大佬教教我，没学过英文，抱歉了，不用英文提问了。

**Comment by Superskyyy** [comment, marked as answer]
Please search for TTL in the docs, it will be deleted once TTL is reached. 

By the way, try using translation service, it will be enough to communicate. I translated for you.

**Reply by MuJianxuan** [reply]
Thank you so much, you said that there is an expired configuration in application.yml, right?

**Reply by Superskyyy** [reply]
> Thank you so much, you said that there is an expired configuration in application.yml, right?

https://skywalking.apache.org/docs/main/latest/en/setup/backend/ttl/ yes

**Reply by hieu276** [reply]
> Please search for TTL in the docs, it will be deleted once TTL is reached.
> 
> By the way, try using translation service, it will be enough to communicate. I translated for you.

can i delete it manually?
